adapters/handlers/graphql/local/get/class_builder_references_test.go

```diff
@@ -25,7 +25,7 @@ func TestGetNoNetworkRequestIsMadeWhenUserDoesntWantNetworkRef(t *testing.T) {
 	resolver := newMockResolver()
 
 	expectedParams := traverser.GetParams{
-		Kind:      kind.Thing,
+		Kind:      kind.Object,
```

Comment:
This seems pointless, as a kind could never be anything but a `kind.Object`. However, I think we can scope this cleanup out to a follow up issue and just consider this tech debt for now?

Reply:
Yes, I have created that `kind.Object` because it was much simpler to remove T/A and replace it with Object. I would create also a separate issue for removing the `kind.Object`